.spectral.yaml

```diff
@@ -102,3 +102,11 @@ rules:
       function: pattern
       functionOptions:
         notMatch: 'string'
+
+  schema-name-length:
```

Comment:
I think we can have a better name which indicates why its being flagged:
something like `schema-name-too-long` or `schema-name-must-be-shorter`?

Reply:
Good points